api/auth.go

```diff
@@ -65,7 +66,20 @@ func (auth *Auth) CreateCustomerJWT(customerId int) ([]byte, error) {
 	return jwt.Sign(t, jwa.ES256, auth.sessionKey)
 }
 
-func (auth Auth) GetCustomerIDFromHeader(ctx echo.Context) (int, error) {
+func (auth *Auth) GetSessions() map[string]Session {
```

Comment:
what about expired sessions?

Reply:
Currently, we don't deal with expires sessions. You have to make sure to do everything within 15 minutes 😅

Reply:
(that was already the case)